crates/bevy_math/src/primitives/mod.rs

```diff
@@ -6,6 +6,13 @@ mod dim2;
 pub use dim2::*;
 mod dim3;
 pub use dim3::*;
+mod frustum;
+pub use frustum::Frustum;
+mod half_space;
+pub use half_space::HalfSpace;
+/// Clean me up!
```

Comment:
This needs real docs explaining why this exists, and linking to an issue made to track their clean-up.